pkg/agent/utils.go

```diff
@@ -65,17 +65,12 @@ var TranslatedKubeletConfigFlags = map[string]bool{
 	"--fail-swap-on":                      true,
 	"--container-log-max-size":            true,
 	"--container-log-max-files":           true,
-	"--serialize-image-pulls":             true,
```

Comment:
dont we wanna keep this?

Reply:
eh ill just add it in the custom kubelet config PR and do a clean revert here.